### PR TITLE
CBAPI-2204: Fix example scripts

### PR DESCRIPTION
**Pull request by emitreva-cb**
## Pull request checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [ ] Tests have been added that prove the fix is effective or that the feature works.
- [x] New and existing tests pass locally with the changes.
- [x] Code follows the style guidelines of this project (PEP8, clean code, linter).
- [x] A self-review of the code has been done.

## What is the ticket or issue number?
https://jira.carbonblack.local/browse/CBAPI-2206

## Pull Request Description
The scripts from https://github.com/carbonblack/cbapi-python/tree/master/examples/defense/cblr were moved to examples/live_response and updated to work.
live_response_cli.py updated to work with the new Device model and v6 of the liveresponse API.

## Does this introduce a breaking change?

- [ ] Yes
- [ ] No

## How Has This Been Tested?
Manual testing was done to verify that it works correctly.

### CODE REVIEW

File: examples/live_response/examplejob.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+# * DISCLAIMER. THIS PROGRAM IS PROVIDED TO YOU "AS IS" WITHOUT
+# * WARRANTIES OR CONDITIONS OF ANY KIND, WHETHER ORAL OR WRITTEN,
+# * EXPRESS OR IMPLIED. THE AUTHOR SPECIFICALLY DISCLAIMS ANY IMPLIED
+# * WARRANTIES OR CONDITIONS OF MERCHANTABILITY, SATISFACTORY QUALITY,
+# * NON-INFRINGEMENT AND FITNESS FOR A PARTICULAR PURPOSE.
+
+"""Example script for get file"""
+
+
+class GetFileJob(object):
```
Comment (avanbrunt-cb) [inline examples/live_response/examplejob.py:16]: Would be great to show that a job can do more than one command. Can we add a `list directory` command of the directory where the file is located and print the results to stdout then pass the file back?

**Review by avanbrunt-cb — COMMENTED** [review]
Overall, looks great just one suggestion to make the example job a little better of an example

File: examples/live_response/examplejob.py (hunk 1 above)
Comment (emitreva-cb) [reply examples/live_response/examplejob.py:16]: Done.

**Review by avanbrunt-cb — APPROVED** [review]

**Comment by hkaragitliev-cb** [comment]
LGTM, only one nit-picky thing - I`m not very fond of mixing f strings and the .format without a reason. Either one is fine, but both seem to a bad styling.

**Review by hkaragitliev-cb — APPROVED** [review]

**Pull request merged**